GameFighter/Battle.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace GameFighter
+{
+    public static class Battle
+    {
+        public static bool Fight(Warrior warrior1, Warrior warrior2)
+        {
+            if (!warrior1.IsAlive || !warrior2.IsAlive)
```

Comment:
What if any parameter is null?
Add curly braces for _if_ statements (here and below)